spec/models/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe User, :type => :model do
+  it "has role" do
+    expect(User.new(role: "Standard")).to have_attributes(role: "Standard")
+  end
+
+  it "has standard role by default" do
+    expect(User.new).to have_attributes(role: "Standard")
+  end
+
+  it "has role set to standard" do
+    user = User.new(role: "Standard")
+    expect(user.standard?).to eq(true)
+  end
+
+  it "has role set to premium" do
+    user = User.new(role: "Premium")
+    expect(user.premium?).to eq(true)
+  end
+
+  it "has role set to admin" do
+    user = User.new(role: "Admin")
+    expect(user.admin?).to eq(true)
+  end
+end
```

Comment:
Currently these tests would pass if the implementation for the `admin?` etc. methods looked like this:

```ruby
  def admin?
    true
  end

  def standard?
    true
  end

  def premium?
    true
  end
```

Expand this spec to include test cases for:

- admin? returns false for non-admin
- premium? returns false for non-premium
- standard? returns false non-standard